pkg/conf/config.go

```diff
@@ -35,3 +41,35 @@ func GetBuilderKey() (string, error) {
 	builderKey := string(builderKeyBytes)
 	return builderKey, nil
 }
+
+func GetStorageParams(env sys.Env) (Parameters, error) {
```

Comment:
I kept in conf because i am getting the storage params from the environment configuration.